src/Waypoint/Directive.js

```diff
@@ -8,8 +8,10 @@ var zumWaypoint = function zumWaypoint(WaypointService) {
 			waypoints : '=?zumWaypoint'
 		},
 		link : function zumWaypointLink(scope, element, attrs, ctrl) {
-			var callback = $.proxy(ctrl.processWaypoint, ctrl);
-			element.waypoint({
+			var callback = angular.bind(ctrl, ctrl.processWaypoint);
+			/*jshint -W031 */
```

Comment:
@jonyo Small thing:  you should add a jshint directive to turn this hint back on after the constructor below.